init.lua

```diff
@@ -5,48 +5,48 @@
 vim.g.mapleader = " "
 vim.g.maplocalleader = " "
 
-
 --
 -- NeoVim Options
 -- :help options
 --
 
 -- Tabstop Settings
-vim.opt.tabstop = 2      -- width of tab character
-vim.opt.softtabstop = 2  -- amount of white space to be added
-vim.opt.shiftwidth = 2   -- amount of white space to add in normal mode
+vim.opt.tabstop = 2 -- width of tab character
+vim.opt.softtabstop = 2 -- amount of white space to be added
+vim.opt.shiftwidth = 2 -- amount of white space to add in normal mode
 vim.opt.expandtab = true -- use spaces instead of tabs
 
 -- Indentation Settings
 vim.opt.smartindent = true -- autoindenting when starting a new line
-vim.opt.wrap = false       -- disable line wrapping
+vim.opt.wrap = false -- disable line wrapping
 
 -- Search
-vim.opt.incsearch = true  -- enable incremental search
+vim.opt.incsearch = true -- enable incremental search
 vim.opt.ignorecase = true -- ignore case in search pattern
-vim.opt.smartcase = true  -- case sensitive, if search pattern contains upper case characters
+vim.opt.smartcase = true -- case sensitive, if search pattern contains upper case characters
 vim.opt.hlsearch = true -- disable highlighting
 
 -- Appearance
 vim.opt.relativenumber = true -- enable relative line numbers
-vim.opt.colorcolumn = '120'   -- highlighted line length
-vim.opt.signcolumn = "yes"    -- draw the signcolumn (default = "auto")
-vim.opt.cmdheight = 1         -- number of screen lines to use for the command-line
-vim.opt.scrolloff = 10        -- minimal number of screen lines to keep above and below the cursor
+vim.cmd([[set nu]]) -- show real line number for current line
```

Comment:
Is that different to `vim.opt.number = true`?

Reply:
Anyway, I have take it over. 👍

Reply:
You are right. It is actually the same. I changed it.